activemodel/lib/active_model/serializers/json.rb

```diff
@@ -42,6 +42,13 @@ module JSON
       #   # => { "user" => { "id" => 1, "name" => "Konata Izumi", "age" => 16,
       #   #                  "created_at" => "2006-08-01T17:27:13.000Z", "awesome" => true } }
       #
+      # If you prefer to use custom name than the object's type, set <tt>:root</tt> to the custom name as in:
```

Comment:
I would say `If you prefer, <tt>:root</tt> may also be set to a custom string key:` instead. `as_json(root: :author)` would make the root a symbol which I don't think we want. This might also signal room for improvement in the API since we are expecting keys to be strings here, and not calling `to_s` on custom roots.

Reply:
Reworded https://github.com/rails/rails/pull/36092/commits/d8c5b08714d6f97c0726a8eca3a454d85f4f59d1

>as_json(root: :author) would make the root a symbol which I don't think we want. 

It makes root a string key, not symbol.

Reply:
https://github.com/rails/rails/blob/8365b7ded693084f84588416657c932effb5ff74/activemodel/lib/active_model/serializers/json.rb#L99

It doesn't call `root.to_s` on this line so calling `user.as_json(root: :author)` would result in a symbol root key. Unless I'm misunderstanding the intent of this patch, please update the example code to use a string.

Also, please squash you commits into one, and then I'll merge this. Thanks! 🙏

Reply:
@gmcgibbon  I was just deceived by what `render json` was doing. But I was able to see what you were saying. My bad. 

I have made the change and squashed my commit. Hope it's good now.